prpr/src/judge.rs

```diff
@@ -472,13 +472,8 @@ impl Judge {
                     {
                         continue;
                     }
-                    let dt = dt
-                        + if matches!(note.kind, NoteKind::Flick | NoteKind::Drag) {
-                            0.05 /* TODO tweak*/
-                        } else {
-                            0.
-                        };
-                    if dt + (dist / res.note_width - 1.).max(0.) * DIST_FACTOR < closest.2 + (closest.1 / res.note_width - 1.).max(0.) * DIST_FACTOR {
+                    let dt = if matches!(note.kind, NoteKind::Flick | NoteKind::Drag) {dt.max(LIMIT_PERFECT)} else {dt};
+                    if dt + (dist / res.note_width - 1.).max(0.) * DIST_FACTOR < closest.2 - 0.01 + (closest.1 / res.note_width - 1.).max(0.) * DIST_FACTOR {
```

Comment:
0.01 here needs more tests, I'll resolve this later

Reply:
I've tested a main situation.
At least for this one:
![image](https://user-images.githubusercontent.com/87891461/219258729-b287fda6-af45-45ff-9f5f-795762057d46.png)
it works well.

Reply:
Btw I dont think 10ms has an important impact. There are 20ms intervals even between two 48th notes in 180 bpm.